src/fairseq2/models/_hub.py

```diff
@@ -163,6 +164,57 @@ def load(
 
         return cast(ModelT, model)
 
+    def load_from_path(
```

Comment:
should it be already copied from fs2 (nothigh specific to mms)

Reply:
Is already an added interface in the upcoming rc `load_model`. Can be omitted.